ormconfig.ts

```diff
@@ -1,17 +1,18 @@
 import { TypeOrmModuleOptions } from '@nestjs/typeorm';
 import { readFileSync } from 'fs';
 const serverCa = [readFileSync("./DigiCertGlobalRootCA.crt.pem", "utf8")];
+console.log(process.env.MYSQL_HOST)
 export const ormConfig: TypeOrmModuleOptions = {
   type: 'mysql',
   host:  process.env.MYSQL_HOST,
   port: Number(process.env.MYSQL_PORT),
   username: process.env.MYSQL_USER,
   password: process.env.MYSQL_PASSWORD,
   database: process.env.MYSQL_DATABASE,
-  ssl: {
-    rejectUnauthorized:true,
-    ca: serverCa,
-  },
```

Comment:
Why did you comment out those lines? Did you encounter any problems with this database configuration/connection? If not, please uncomment them before merging to the develop branch.

Reply:
I am using local db without SSL, will uncomment before pushing